### PR TITLE
Polkadot v0.9.24

### LINKED ISSUE #60: substrate dependencies for `polkadot-v0.9.24` branch
Hi, would we have a `polkadot-v0.9.24` branch for newer substrate dependencies?

### PR DESCRIPTION
**Pull request by haerdib**
closes #60

### CODE REVIEW

**Review by clangenb — APPROVED** [review]
Lgtm!

File: teerex/Cargo.toml [hunk 1]
```diff
@@ -12,34 +12,34 @@ edition = "2021"
 codec = { version = "3.0.0", default-features = false, features = ["derive"], package = "parity-scale-codec" }
 log = { version = "0.4.14", default-features = false }
 scale-info = { version = "2.0.1", default-features = false, features = ["derive"] }
-serde = { version = "1.0.101", features = ["derive"], optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
```
Comment (gaudenzkessler) [inline teerex/Cargo.toml:15]: different version (1.0.13) correct?

**Review by gaudenzkessler — APPROVED** [review]
Just a question

File: teerex/Cargo.toml (hunk 1 above)
Comment (haerdib) [reply teerex/Cargo.toml:15]: Yes, exactly. Removed the very strict versioning altogether, we should take the same as the polkadot branch anyway.

**Pull request merged**